App.js

```diff
@@ -4,8 +4,9 @@ import React, { useEffect, useState } from 'react';
 import * as Sentry from 'sentry-expo';
 
 import { MainApp } from './src';
-import { fontConfig, sentryApi } from './src/config';
+import { fontConfig, namespace, secrets } from './src/config';
 
+let sentryApi = secrets[namespace].sentryApi;
```

Comment:
`let`? `const` here, right?

Reply:
Yes definitely. I probably was still in Swift-mode 😅